qemu/tests/netuser_buffer_test.py

```diff
@@ -0,0 +1,72 @@
+import logging
+import os
+import time
+
+from avocado.utils import process
+from virttest import error_context
+from virttest import data_dir
+from virttest import utils_net
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    1) Boot guest with "-net user" and virtio-net backend
+    2) Set MTU value in guest
+    3) Compile the script and execute
```

Comment:
Should we add the steps description about the checking after executing the script?

Reply:
Thank you for your reminder, I have added relevant instructions. Please help me review again, thanks.